packages/@vue/cli-service/lib/options.js

```diff
@@ -8,6 +8,7 @@ const schema = createSchema(joi => joi.object({
   filenameHashing: joi.boolean(),
   runtimeCompiler: joi.boolean(),
   transpileDependencies: joi.array(),
+  skipPlugins: joi.array(),
```

Comment:
Seems redundant?

Reply:
Oh right, that's a leftover from my initial attempt to solve it through a config option instead of a flag.